package.json

```diff
@@ -68,14 +68,9 @@
   "simple-git-hooks": {
     "pre-commit": "npx next-hashicorp precommit"
   },
-  "workspaces": [
-    "./packages/*"
-  ],
+  "workspaces": ["./packages/*"],
```

Comment:
Not sure why these formatting changes are happening, haven't tried messing with this too much but happy to debug further if we feel it's a priority.